plugins/lime-plugin-pirania/src/pages/create.js

```diff
@@ -14,10 +14,11 @@ import Loading from '../../../../src/components/loading';
 import daysFromNow from '../../../../src/utils/daysFromNow';
 import makeid from '../../../../src/utils/makeid';
 
-function Create({ goBack, createMemberVoucher, createVisitorVoucher, daysLeft, createVoucher, loading, createEpoc, list }) {
+function Create({ goBack, createMemberVoucher, createVisitorVoucher, createVoucher, loading, createEpoc, list }) {
+	const initialNote = 'marcos android';
```

Comment:
maybe a more sensible value, one that could hint the user to what to put there?

Reply:
That's what I imagine users putting there. What were u imagining?

Reply:
Dunno!